_layouts/default.html

```diff
@@ -7,307 +7,388 @@
-                </div>
-              </div>
-            </div>
-          </div>
+    <div class="flex" aria-label="Content">
+      <div class="container herounit">
+        <div class="flex flex--vertical content">
+          <p>
+            Cylc is a <b>general purpose workflow engine</b> that also
+            orchestrates <b>cycling systems</b> very efficiently.
+            It is used in production weather, climate, and environmental
+            forecasting on HPC, but is not specialized to those domains.
+          </p>
+
+          <p>
+            Cylc is an active Open Source project
+            <a href="https://github.com/cylc">on GitHub</a>.
+            <br/>
+            See <a href="#publications">Cylc Publications and Reports</a> below.
+          </p>
         </div>
 
-
-        <div class="flex" aria-label="Content">
-          <div class="container herounit">
-
-            <div class="flex flex--vertical content">
-
-              <p>
-              Cylc is a <b>general purpose workflow engine</b> that also
-              orchestrates <b>cycling systems</b> very efficiently.
-              It is used in production weather, climate, and environmental
-              forecasting on HPC, but is not specialized to those domains.
-              </p>
-
-              <p>
-              Cylc is an active Open Source project
-              <a href="https://github.com/cylc">on GitHub</a>.
-              <br/>
-              See <a href="#publications">Cylc Publications and Reports</a> below. 
-              </p>
-            </div>
-
-            <div class="flex flex--vertical release">
-              <div class="flex flex--vertical">
-                <div class="flex download-buttons">
-
-                  <div class="flex flex--vertical m-r">
-                    <label>Latest (Python 3)</label>
-                    <a class="flex grow" id="cylc8-button">
-                      <button class="button flex grow">
-                        <img src={{'/assets/icon-download.svg' | relative_url}} />
-                        <span id="cylc8-latest-release-name"></span>
-                      </button>
-                    </a>
-                    <label id="cylc8-latest-release-pubdate-text"></label>
-                  </div>
-
-                  <div class="flex flex--vertical m-l">
-                    <label>Legacy (Python 2)</label>
-                    <a class="flex grow" id="cylc7-button">
-                      <button class="button flex grow">
-                        <img src={{'/assets/icon-download.svg' | relative_url}} />
-                        <span id="cylc7-latest-release-name"></span>
-                      </button>
-                    </a>
-                    <label id="cylc7-latest-release-pubdate-text"></label>
-                  </div>
-                </div>
-
-                <h3 class="flex jc-center">Next Release:&nbsp;
-                  <a class="page-link" id="milestone-url" target="_blank"><span
```

Comment:
Removed the download buttons. They don't seem to work and we're beyond the days of tarballs.